### ISSUE TITLE
Are there any Lovelace cards that work with Ecowitt?

### ISSUE THREAD

**Issue by fun4stuff** [garbled1/homeassistant_ecowitt#64]
The one recommended In the read me no longer works.

**Comment by DaHype** [comment]
Is there any way to get the Dashboard [ecowitt.net](http://ecowitt.net/) into Home Assistant?
Sure i can create an Webpage card but i have to insert login in the Card, and that don´t works.
There only flash the card, because i cant log in.

Any Idea?

Thanks

**Comment by godfreym29** [comment]
Just use the Ecowitt Weather Station add-on in HACS

**Comment by DaHype** [comment]
Of course. I have it and use it already, but i want to have the Dashboard from the Ecowitt with Graphic, Design.....

I can create an Webpage card from my Station. But i have to insert for it Password and User, and that don´t work with the Webpage card.
